test/e2e/storage/drivers/in_tree.go

```diff
@@ -1605,6 +1606,12 @@ func (a *azureDiskDriver) PrepareTest(f *framework.Framework) (*testsuites.PerTe
 	}, func() {}
 }
 
+func (a *azureDiskDriver) GetTimeouts() framework.TimeoutContext {
+	t := framework.NewTimeoutContextWithDefaults()
+	t.PodStart = time.Minute * 10
+	return t
+}
```

Comment:
This (custom timeouts in Azure Disk) is just an example of how to add custom timeouts to an in-tree driver. I plan to remove it before merging this PR.